### PR TITLE
Merge Conflict Resolution (from 7.7.x to master)

### PR DESCRIPTION
**Pull request by ConfluentSemaphore**
# What is this pull request?

This is an autogenerated pull request by `pint` due to a merge conflict failure seen during [`pint merge`](http://go/pint-merge-service).

# Next Steps

1. Resolve all conflicts (either via web editor or the command line)
2. Commit merge
3. Obtain approvals from reviewers
4. **(IMPORTANT)** After obtaining approvals, click the dropdown menu on the merge button and select **"Create a merge commit"** 
5. Click the "Merge pull request" button
6. Re-run [`pint merge`]( http://go/pint-merge-service)

NOTE: **"Create a merge commit"** option must be used to ensure all commits from the head branch is added to the base branch.

# Questions?
Please reach out to the Developer Productivity team on Slack at #devprod-eng or #devprod-oncall

### CODE REVIEW

**Review by davetroiano — APPROVED** [review]

**Review by sandonjacobs — APPROVED** [review]

**Comment by davetroiano** [comment]
@brianstrauch I don't know why this says `Waiting on code owner review from confluentinc/cli` when `developer-advocates` is a CODEOWNER and sandon (a member) approved.... but can you please give this a ➕ ?

File: clients/docs/includes/configs/cloud/java.config
Comment (brianstrauch) [inline clients/docs/includes/configs/cloud/java.config:12]:
```suggestion
# Best practice for Kafka producer to prevent data loss
```

File: clients/docs/includes/configs/cloud/java-sr.config
Comment (brianstrauch) [inline clients/docs/includes/configs/cloud/java-sr.config:12]:
```suggestion
# Best practice for Kafka producer to prevent data loss
```

**Review by brianstrauch — APPROVED** [review]
Thanks!

**Pull request merged**
